documentation/modules/upgrading/proc-downgrade-brokers-older-kafka.adoc

```diff
@@ -8,30 +8,11 @@
 
 This procedure describes how you can downgrade a Strimzi Kafka cluster to a lower (previous) version of Kafka, such as downgrading from {KafkaVersionHigher} to {KafkaVersionLower}.
 
-[IMPORTANT]
-====
-Downgrading is _not possible_ if the new version has ever used a `log.message.format.version` that is not supported by the previous version, including when the default value for `log.message.format.version` is used. For example, this resource can be downgraded to Kafka version {KafkaVersionLower} because the `log.message.format.version` has not been changed:
-
-[source,yaml,subs=attributes+]
-----
-apiVersion: v1alpha1
-kind: Kafka
-spec:
-  # ...
-  kafka:
-    version: {KafkaVersionHigher}
-    config:
-      log.message.format.version: "{LogMsgVersLower}"
-      # ...
-----
-
-The downgrade would not be possible if the `log.message.format.version` was set at `"{LogMsgVersHigher}"` or a value was absent (so that the parameter took the default value for a {KafkaVersionHigher} broker of {LogMsgVersHigher}).
-====
-
 .Prerequisites
 
 For the `Kafka` resource to be downgraded, check:
 
+* [IMPORTANT] xref:com-target-downgrade-version-{context}[Compatibility of Kafka versions].
```

Comment:
I'm not sure about the square bracket convention here. Perhaps bold is better?

Reply:
I tried the bold, but thought just the capital was enough in the end. We use a similar format elsewhere, like with _ADVANCED OPTION:_ for callouts

![image](https://user-images.githubusercontent.com/47596553/99826905-e0926280-2b50-11eb-83d1-6a95f8ec5224.png)